source/server/configuration_impl.h

```diff
@@ -74,6 +74,13 @@ class FilterChainUtility {
    */
   static bool buildFilterChain(Network::ListenerFilterManager& filter_manager,
                                const std::vector<Network::ListenerFilterFactoryCb>& factories);
+
+  /**
+   * Given a UdpListenerFilterManager and a list of factories, create a new filter chain. Chain
+   * creation will exit early if any filters immediately close the connection.
+   */
+  static bool buildFilterChain(Network::UdpListenerFilterManager& filter_manager,
```

Comment:
nit: rename to buildUdpFilterChain instead of overloading TCP one to make it easier for code search.

Reply:
ack